index.html

```diff
@@ -0,0 +1,245 @@
+             <button class="navbar-toggler d-md-none d-block" type="button" data-bs-toggle="collapse" data-bs-target="#main-nav" aria-controls="main-nav" aria-expanded="false" aria-label="Toggle Navigation">
+                <span onclick="ShowXicon()" class="navbar-toggler-icon" id="meneIcon"></span>
+                <span onclick="removeXiocn()" class="bi bi-x" id="Xicon"></span>
+            </button>
+            
+            <a href="#intro" class="navbar-brand">
+                <span class="logo">
+                  <a href="index.html"><img class="logo" src="assets/images/logo/techlife.png" alt=""></a> 
+                </span>
+            </a>
+
+           
+            <!-- Navbar links -->
+            <div class="collapse navbar-collapse justify-content-end pe-5 pt-1 align-center nav-menu" id="main-nav">
+                <ul class="navbar-nav">
+                    <li class="nav-item pt-1">
+                        <a class="nav-link" href="about.html">About</a>
+                    </li>
+                    <li class="nav-item pt-1">
+                        <a class="nav-link" href="#main-program">Program</a>
+                    </li>
+                    <li class="nav-item pt-1">
+                        <a class="nav-link" href="#speakers">Join</a>
+                    </li>
+                    <li class="nav-item pt-1">
+                        <a class="nav-link" href="#partners">Sponsor</a>
+                    </li>
+                    <li class="nav-item pt-1">
+                        <a class="nav-link" href="#">News</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">CC Campaign</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <!-- main page  -->
+    <section id="intro">
+        <div class="container-lg pt-5 background-gradient">
+            <div class="row justify-content-center align-items-center">
+                <div class="col-md-8 col-md-start">
+                    <h1>
+                        <div class="display-6 px-4"> "Hello! Sharing World"</div>
+                        <div class="display-5 px-3">Techlife Innovative global summit 2023</div>
+                    </h1>
+                    <p class="lead-p px-3 py-2 col-md-9 col-md-start">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+
+                    <div class="col-md-8 pt-3 date-p-show col-md-start">
+                        <div class="date-show px-3 col-md-10 col-md-start">2023.07.04(TUE) ~ 14(FRI)</div>
+                        <p class="date-p px-3 py-4 col-md-10 col-md-start">@ National Museum of Karea, Art Center Nabi and more
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] As required for the project to stick as much as possible to the one form the Behance template, Kindly ensure that you improve the design of your here section as right now it doesn't match the one from the given template e.g.: fonts(size, weight,...), padding, height, margin, text-alignment,...
### Template design
![Screenshot from 2023-07-06 18-49-54](https://github.com/MozamelJawad/Capstone-Project/assets/83042742/17e06ca1-5fc4-4cec-8870-cb6075db7117)
### Your Design
![Screenshot from 2023-07-06 19-00-16](https://github.com/MozamelJawad/Capstone-Project/assets/83042742/7345e2cd-0f3a-46c2-8907-a425bc81e810)